src/pages/Contact/styles/Contact.styled.ts

```diff
@@ -50,8 +51,8 @@ export const SectionContent = styled.div`
 export const ContactLink = styled.a`
   font-style: normal;
   font-weight: 300;
-  font-size: clamp(1vw, 2vw, 30px);
-  color: inherit;
+  font-size: clamp(1vw, 2vw, 24px);
```

Comment:
why the min using vw, usually min n max are absolute values; also can this use values from global styles?

Reply:
Resolved in 9fd696b

Added padding left to ensure space between each contact link